lib/sycamore/sycamore/connectors/duckdb/duckdb_writer.py

```diff
@@ -3,8 +3,8 @@
 from typing_extensions import TypeGuard
 
 from sycamore.data.document import Document
-from sycamore.connectors.writers.base import BaseDBWriter
-from sycamore.connectors.writers.common import convert_to_str_dict
+from sycamore.connectors.writer_utils.base import BaseDBWriter
+from sycamore.connectors.writer_utils.common import convert_to_str_dict
```

Comment:
maybe let's just move these files to sycamore/connectors instead of sycamore/connectors/writer_utils/

Reply:
We'll want to create separate reader utils as well, so I think it would be better to have this organization. At the very least we should rename the files to make it clear they are writer files